.github/workflows/system.yml

```
@@ -0,0 +1,28 @@
+---
+name: System Tests
+
+on:
+  pull_request:
+
+jobs:
+  system:
+    name: Test
```

Unfortunately only the job name is displayed in the branch settings tab so having that more descriptive helps. I suggest job name "_Deployment Test_", so full name "_System Tests/Deployment Test_".

```suggestion
    name: Deployment Test
```